docs/source/distributed.tensor.rst

```diff
@@ -145,6 +145,47 @@ specifying the :class:`DeviceMesh` and :class:`Placement` for the :class:`DTenso
 
 .. autofunction:: randn
 
+DTensor Semantics
+------------------
+
+Uneven Sharding
+^^^^^^^^^^^^^^^
+DTensors can only represent a subset of all possible sharding configurations.
+
+A sharding for a tensor dimension T over a mesh dimension M is legal if it satisfies the following conditions:
+- If M evenly divides T, the sharding shall be 'even' (local shards will have the same size T/M on each rank)
+- Otheriwse, following torch.chunk semantics, mesh-dim-local ranks 0..M-2 will get chunks of size T/M, and rank M-1 will
```

Comment:
The description of `M-2` and `M-1` is not accurate. Thinking of the following example:
```
T = 12
M = 8
```
The first 6 ranks get 2 each, the last 2 ranks get 0.

Besides, to be more concisely, `ceil(T // M)` rather than `T/M`.